src/svg/reddit.svg

```diff
@@ -0,0 +1 @@
2.238C4.71 221.976.533 210.487 0 198.314V198.09c.013-17.068 7.424-33.228 18.945-45.11 11.52-11.95 27.252-19.743 44.136-19.743h.7c1.758-.12 3.557-.172 5.342-.172 8.478 0 16.976 1.364 25.045 4.493h.086c6.89 2.904 13.734 6.43 19.834 11.265 1.977-1.02 4.13-2.22 6.555-3.13 36.982-21.902 79.958-30.32 121.848-32.766.362-20.764 2.91-42.495 13.03-61.5 8.425-15.81 23.05-28.102 40.48-32.37h.157c6.627-1.305 13.32-1.877 19.96-1.877 17.706 0 35.19 4.156 51.565 10.698 7.233-10.922 17.575-19.46 29.71-24.235l.118-.06.105-.053C404.73 1.31 412.267 0 419.83 0c7.956 0 15.98 1.482 23.503 4.835v-.053l.026.053.13.06c10.79 4.157 19.92 11.83 26.39 21.33 6.47 9.618 10.29 21.105 10.29 32.884 0 2.16-.132 4.32-.383 6.54l-.013.06-.026.112c-1.265 15.25-8.985 28.616-19.92 38.115-11 9.617-25.243 15.42-39.867 15.42-2.345 0-4.716-.17-7.06-.46-14.507-1.134-28.024-8.13-37.958-18.604-9.974-10.408-16.39-24.347-16.39-39.36 0-.685.066-1.48.093-2.166-13.623-6.087-28.062-11.094-42.437-11.094-2.068 0-4.15.12-6.23.343h-.027c-10.75 1.027-20.75 7.793-25.598 17.522v.053c-7.365 14.342-8.814 31.067-9.038 47.733 41.224 2.675 82.71 12.575 119.087 33.45h.054l.527.343c.686.396 1.95 1.08 3.123 1.707 2.49-2.05 5.14-4.098 8.155-5.916 11.027-7.338 24.084-10.98 37.153-10.98 5.81 0 11.62.737 17.26 2.16h.013l.225.06.224.06c13.307 3.41 25.36 11.033 34.426 21.33 9.05 10.354 15.15 23.384 16.337 37.6zm-345.39 80.38h.15c3.254 1.198 6.606 1.765 9.934 1.765 8.286 0 16.342-3.347 22.442-8.866 6.093-5.573 10.257-13.426 10.257-22.358l-.015-.737.033-1.316c0-8.867-4.11-16.785-10.145-22.292-6.054-5.64-14.09-9.05-22.41-9.05-2.22 0-4.493.288-6.7.796h-.125c-11.008 2.34-20.678 10.467-24.24 21.73l-.015.067c-1.02 3.057-1.495 6.244-1.495 9.38 0 6.93 2.272 13.597 6.198 19.17 3.887 5.4 9.4 9.67 16.008 11.66m179.218 41.42c-2.635-1.713-5.665-2.74-8.88-2.74-2.582 0-5.27.737-7.654 2.345-22.62 13.48-49.234 20.87-75.644 20.87-19.82 0-39.524-4.15-57.37-12.912l-.092-.053-.085-.053c-2.688-1.08-5.758-3.426-9.104-5.64-1.7-1.132-3.47-2.213-5.44-3.017-1.943-.855-4.104-1.423-6.41-1.423-1.91 0-3.92.462-5.902 1.305l-.302.118h.013c-3.107 1.2-5.538 3.413-7.106 5.97-1.634 2.687-2.438 5.743-2.438 8.826 0 2.794.66 5.574 2.016 8.077 1.29 2.385 3.274 4.492 5.876 5.862 25.514 17.232 55.545 25.03 85.702 24.966 27.187 0 54.538-6.258 78.885-17.747l.132-.054.132-.052c3.2-1.95 7.47-3.65 11.487-6.087 2.003-1.317 3.94-2.74 5.612-4.678 1.66-1.87 3.003-4.203 3.767-6.996.304-1.252.462-2.45.462-3.584 0-2.266-.567-4.427-1.49-6.377-1.373-2.844-3.585-5.215-6.168-6.928zm20.473-82.5c1.318 3.372 1.91 6.89 1.91 10.354 0 6.942-2.41 13.78-6.455 19.287-3.992 5.468-9.657 9.67-16.323 11.555l-.146.053h-.09c-3.11 1.08-6.325 1.594-9.487 1.594-7.457 0-14.756-2.676-20.605-7.273-5.797-4.56-10.197-11.093-11.567-18.84h-.012l-.014-.105-.026-.065h.026c-.435-1.99-.658-3.98-.658-5.97 0-6.718 2.292-13.2 6.073-18.496 3.767-5.342 9.076-9.55 15.295-11.712h.093c3.518-1.31 7.232-1.99 10.896-1.99 6.692 0 13.28 2.043 18.84 5.745 5.48 3.637 9.934 8.985 12.173 15.65l.053.106.04.104h-.015z"/></svg>
```

Comment:
they shouldn't change :o I mean they could be prettified considering they're transformed into an encoded string